sklearn/decomposition/pca.py

```diff
@@ -353,6 +353,16 @@ def fit_transform(self, X, y=None):
         -------
         X_new : array-like, shape (n_samples, n_components)
 
+        Notes
+        -----
+        This method returns an f-ordered array.
+        To get the output array in c-order,
```

Comment:
I don't think this is what we want. We don't want to take the transpose here because this is only linked to the memory layout. I would phrase it as:

```python
    """
    Notes
    -----
    Transforming `X` through `fit_transform` and `transform` will return a
    Fortran-ordered array. You can convert it to a C-ordered array using
    `np.ascontiguousarray`
    """
```